README.md

```diff
@@ -19,6 +19,24 @@ Once setup has been completed, you can start the server with
 script/server
 ```
 
+### Notes from a local install
+
+We additionally needed to take the following steps to get a reasonable local install outside of Docker including
+a working database for running the server locally:
+
+#### Database
+
+* Install Cloud Foundry from [GPaaS](https://www.cloud.service.gov.uk/) after logging in.
+We needed to open it in Finder to be able to install it (a second button appears in grey)
+* `cf install-plugin conduit`
+* `cf login -a api.london.cloud.service.gov.uk -u your@account.name`
+* `script/db-restore`
+
+#### Tests
+
+* `RAILS_ENV=test bundle exec rake db:create` to create a test database
+* `bundle exec rake assets:precompile` (some tests failed without this step)
```

Comment:
This isn't something we should have to do in Rails dev (normally it's just for production – in dev because you're changing CSS and JS regularly, precompile is generally only something you do to get to the bottom of a problem on production). Might see if I can sit in on James's setup and see if it happens again and if we can work out why certain things aren't being fallen back on; it might be a problem with `environments/test.rb`, but also I suspect that we did our  `db-restore` and had it fail before it set the db `environment`, which broke the ability to create the test db via `db:test:prepare` (it would normally be automatic).

Reply:
Deleted this.